src/compiler/diagnosticMessages.json

```diff
@@ -811,6 +811,18 @@
         "category": "Error",
         "code": 1249
     },
+    "In ES5 or lower, function declarations are not allowed in block scope in strict mode.": {

```

Comment:
`Function declarations are not allowed within blocks in strict mode when targeting 'ES3' or 'ES5'.`